src/types/FeatureFlags.ts

```diff
@@ -0,0 +1,3 @@
+export type FeatureFlags = {
+  tabBarEnabled: Boolean;
```

Comment:
use primitive `boolean` for the feature flag - the TS type system doesn't force objects to their primitive types, and it's recommended to use primitives